sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysis.java

```diff
@@ -79,6 +79,32 @@ private Runner(SensitivityAnalysisProvider provider) {
             return runAsync(network, factorsProvider, contingenciesProvider, SensitivityAnalysisParameters.load());
         }
 
+        public CompletableFuture<SensitivityAnalysisResult> runAsync(Network network,
+                                                                     String workingStateId,
+                                                                     SensitivityAnalysisParameters parameters,
+                                                                     SensitivityFactorsProvider factorsProvider,
+                                                                     ComputationManager computationManager) {
+            return runAsync(network, workingStateId, factorsProvider, new EmptyContingencyListProvider(), parameters, computationManager);
+        }
+
+        public CompletableFuture<SensitivityAnalysisResult> runAsync(Network network,
+                                                                     String workingStateId,
+                                                                     SensitivityAnalysisParameters parameters,
+                                                                     SensitivityFactorsProvider factorsProvider) {
+            return runAsync(network, workingStateId, parameters, factorsProvider, DefaultComputationManagerConfig.load().createLongTimeExecutionComputationManager());
```

Comment:
I think, if the contingency list is empty we should use the short-time execution computation manager.

@murgeyseb Do you agree?

Reply:
Yes, why not.